src/vcCamera.cpp

```diff
@@ -600,7 +600,7 @@ void vcCamera_HandleSceneInput(vcState *pProgramState, udDouble3 oscMove, udFloa
     memset(isBtnClicked, 0, sizeof(isBtnClicked));
     memset(isBtnDoubleClicked, 0, sizeof(isBtnDoubleClicked));
     memset(isBtnHeld, 0, sizeof(isBtnHeld));
-    //memset(isBtnReleased, 0, sizeof(isBtnReleased)); // 09/04/19 - commented out to fix pan-release bug, check back and review once more fully tested
+    //memset(isBtnReleased, 0, sizeof(isBtnReleased));
```

Comment:
Can you create an issue in Azure for this unless it already exists? I recommend referencing this code block rather than this specific line that's commented out as it may not exist when the next person comes along to fix the pan-release problem.

Reply:
Do we need an issue though? I mean this is pretty much how I'd go about fixing it regardless, when would we ever want to ignore a mouse button release event while the window has mouse focus?

Reply:
If this is the correct solution, then replace it with a meaningful comment about how the `isBtnReleased` variable shouldn't be zeroed.